impl/src/main/java/org/ehcache/impl/internal/statistics/DefaultStatisticsServiceConfiguration.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright Terracotta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.ehcache.impl.internal.statistics;
+
+import org.ehcache.core.spi.service.StatisticsService;
+import org.ehcache.core.spi.service.StatisticsServiceConfiguration;
+
+import java.util.concurrent.TimeUnit;
+
+public class DefaultStatisticsServiceConfiguration implements StatisticsServiceConfiguration {
```

Comment:
Note: I did not pushed further to add a configuration parser to be able to configure that from XML... These values control how many latency samples we want to keep and what is the interval used to compute one sample.
@chrisdennis @anthonydahanne @GaryWKeim : WDYT ?

Reply:
I'm of the opinion that everything you can configure programmatically has an equivalent XML form, but that has not been the attitude of other members. I think for this partial feature we can leave it in code, but when the remaining parts are implemented, we add the XML configuration.